unit_test.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+TEST_RESULT=0
+DOCKERFILE='Dockerfile-test'
+IMAGE='tracker'
+POSTGRES_IMAGE='quay.io/cloudservices/postgresql-rds:15-1'
+TEARDOWN_RAN=0
+
+
+teardown() {
+
+    [ "$TEARDOWN_RAN" -ne "0" ] && return
+
+    echo "Running teardown..."
+
+    docker rm -f "$TEST_CONTAINER_NAME"
+
+    # remove postgres container
+    docker rm -f postgres
+    TEARDOWN_RAN=1
+}
+
+trap teardown EXIT ERR SIGINT SIGTERM
+
+mkdir -p artifacts
+
+get_N_chars_commit_hash() {
+
+    local CHARS=${1:-7}
+
+    git rev-parse --short="$CHARS" HEAD
+}
+
+TEST_CONTAINER_NAME="tracker-$(get_N_chars_commit_hash 7)"
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Pulling postgres image"
+docker pull $POSTGRES_IMAGE
+
+echo -e "\n---------------------------------------------------------------\n"
+
+echo "Starting postgres container"
+docker run --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5 -d --rm --name postgres -e POSTGRES_PASSWORD=crc -e POSTGRES_USER=crc -e POSTGRES_DB=crc -h 0.0.0.0:5432:5432 $POSTGRES_IMAGE
```

Comment:
I think the env var names need to adjusted.
`
 -e POSTGRESQL_PASSWORD=crc -e POSTGRESQL_USER=crc -e POSTGRESQL_DATABASE=crc
`
I think this is causing the image/container to fail to start and that's causing the migration to fail to connect.

I'm running into some issues with podman locally...so I'm struggling to get this working locally but I think we need to start with this change.